tailwind.config.js

```diff
@@ -58,6 +58,11 @@ module.exports = {
         "surface-16dp": lighten(surfaceColor, 15),
         "surface-24dp": lighten(surfaceColor, 16),
       },
+      flexBasis: {
```

Comment:
Not sure about this implementation. :thinking: Is there maybe a better way to do it with built-in classes?

Reply:
I could not find a way to do this with the provided Tailwind CSS classes. That's why I went for this option. The problem was, setting the styles with the provided classes, the gap is not being considered and it breaks early having only 4 in a row and some leftover space on the right side. I also found other people asking for this problem on Stackoverflow.

Reply:
Awesome, thanks for helping out. 👍 I was so focused on the "gap", I forgot about the negative margin trick.